.github/workflows/cd.yml

```diff
@@ -117,6 +117,9 @@ jobs:
   prep-testbed:
     runs-on: ubuntu-latest
     needs: update-schema
+    with:
+      inputs: '{ "release_token": "${{ env.release_token }}", "triggered_by": "CD"}'
```

Comment:
`${{ env.release_token }}` isn't available in the `prep-testbed` job. Please use 
```yaml
env:
    release_token: ${{ secrets.JINA_CORE_RELEASE_TOKEN }}
```